provider/google.go

```diff
@@ -145,16 +145,24 @@ func (p *GoogleProvider) Zones() (map[string]*dns.ManagedZone, error) {
 		for _, zone := range resp.ManagedZones {
 			if p.domainFilter.Match(zone.DnsName) {
 				zones[zone.Name] = zone
+				log.Debugf("Matched %s (zone: %s)", zone.DnsName, zone.Name)
+			} else {
+				log.Debugf("Filtered %s (zone: %s)", zone.DnsName, zone.Name)
 			}
 		}
 
 		return nil
 	}
 
+	log.Debugf("Matching zones against domain filters: %v", p.domainFilter.filters)
 	if err := p.managedZonesClient.List(p.project).Pages(context.TODO(), f); err != nil {
 		return nil, err
 	}
 
+	if len(zones) == 0 {
+		log.Warnf("No zones match domain filters: %v", p.domainFilter.filters)
```

Comment:
Good point.
Similar to `Ttl`, we could add a method to `DomainFilter` for **IsConfigured**.
That would allow us to give more descriptive logs for this.

Thoughts?

Reply:
Sure, what would the logging message ?

Reply:
Updated the PR:
```
WARN[0000] No zones in the project, infra-1, match domain filters: [hello.com]

WARN[0000] No zones found in the project, infra-1
```